assets/app/views/project.html

```diff
@@ -11,38 +11,45 @@
         </div>
       </div>      
       <!-- TODO handle multiple services mapping to the same deploymentConfig/deployment/pod -->
-      <section ng-repeat="(serviceId, servicePodsByLabel) in podsByServiceByLabel" ng-if="services[serviceId]">
+      <section ng-repeat="(serviceId, service) in services">
         <div class="row">
           <div class="col-md-12">
             <div class="tile">
               <h2 class="service">{{serviceId}}
                 <span class="small"> - routing {{services[serviceId].spec.protocol}} traffic on {{services[serviceId].spec.portalIP}}:{{services[serviceId].spec.port}} to port {{services[serviceId].spec.containerPort}}</span>
-                <span class="connector connector-vertical" style="left: 50%; top: 35px; height: 21px;">
+                <span class="connector connector-vertical" style="left: 50%; top: 31px; height: 21px;">
                   <span class="connector-endpoint connector-endpoint-top"></span>
                   <span class="connector-line" style="height: 21px;"></span>
                   <span class="connector-endpoint" style="top: 18px; "></span>
                 </span>
               </h2>
               <div class="separator"></div>
-              <div ng-repeat="(deploymentConfigId, deploymentConfig) in deploymentConfigs" ng-if="!deploymentConfig || services[serviceId].spec.selector.name == deploymentConfig.template.controllerTemplate.replicaSelector.name">
+              <!-- Iterate over deployment configs for this service and look for ones that don't have any deployments for this
+                   service, these are likely new so incoming builds are likely, so lets show these now.
+               -->
+              <div ng-repeat="(deploymentConfigId, deploymentConfig) in serviceDeploymentConfigs[serviceId]" ng-if="!serviceDeploymentsByDeploymentConfig[serviceId][deploymentConfigId] && deploymentConfigs[deploymentConfigId]">
+                <!-- As long as pods created from the deployments created from this deployment config will be routed
+                     to by this service, then we will show things related to triggers that are about to create stuff, like builds.
+                -->                
+                <triggers ng-init="triggers = deploymentConfigs[deploymentConfigId].triggers"></triggers>                
```

Comment:
only by doing a scope, which means i have to pass in _every_ parameter i want to make available to it.  of course now that i look at it, i think triggers may be the only thing that is needed since it gets the build off of the trigger